shared/notifications/index.js

```diff
@@ -494,5 +503,19 @@ export const getTranslatedUINotifications = (
             },
     },
     ///: END:ONLY_INCLUDE_IF
+    [NOTIFICATION_PETNAMES]: {
+      ...UI_NOTIFICATIONS[NOTIFICATION_PETNAMES],
+      title: t('notificationsPetnamesTitle'),
+      description: [
+        t('notificationsPetnamesDescriptionOne'),
+        t('notificationsPetnamesDescriptionTwo'),
+      ],
+      actionText: t('notificationsPetnamesActionText'),
+      date: UI_NOTIFICATIONS[NOTIFICATION_PETNAMES].date
```

Comment:
If the date is currently `null`, do we need this logic here now or can we also hardcode `''`?

Reply:
Done.